### PR TITLE
Add scripts for manual task

### PR DESCRIPTION
**Pull request by ShellyXueHan**
this is to be used when we need to manage sysdig resources outside of the operator.

### CODE REVIEW

File: scripts/main.yaml [hunk 1]
```diff
@@ -4,49 +4,35 @@
...
+    - name: get all alerts
+      include_tasks: tasks/get_all_alerts.yaml
 
-    - name: Get team by name
-      set_fact:
-        team_match: "{{ teams_output.json | json_query(jmesquery)}}"
-      vars:
-        jmesquery: "teams[? name=='{{ sysdig_team_name }}']"
+    # - name: Get a dashboard
+    #   include_tasks: tasks/alerts.yaml
```
Comment (w8896699) [inline scripts/main.yaml:13]: Do we want to leave a comment out code here? Aslo scripts/tasks/alerts.yaml is not been removed

File: scripts/tasks/get_all_alters.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    content: "{{ output }}"
+    dest: output/alerts.json
+
+- name: Get alerts
+  set_fact:
+    alerts_list: "{{ output.json.alerts | json_query(jmesquery) }}"
+  vars:
+    jmesquery: "[].name"
+
+- debug: msg="{{ alerts_list }}"
```
Comment (w8896699) [inline scripts/tasks/get_all_alters.yaml:23]: new line at the end of new files

File: scripts/tasks/set_default_dashboard.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        Content-Type: "application/json"
+  register: container_team_admin_user_token
+
+# Add the dashboard that each team will need
+- name: Set Resource Usage Dashboard
+  uri: 
+    url: "{{ sysdig_api_endpoint }}/api/v3/dashboards/"
+    method: POST
+    headers:
+        Authorization: "Bearer {{ container_team_admin_user_token.json.token.key }}"
```
Comment (w8896699) [inline scripts/tasks/set_default_dashboard.yaml:37]: do we want to revoke the token after its been used?

**Review by w8896699 — APPROVED** [review]

File: scripts/main.yaml (hunk 1 above)
Comment (ShellyXueHan) [reply scripts/main.yaml:13]: yea i keep them there for different tasks, just so I can uncomment and run them when needed.

File: scripts/tasks/get_all_alters.yaml (hunk 2 above)
Comment (ShellyXueHan) [reply scripts/tasks/get_all_alters.yaml:23]: there is a new line, no?

File: scripts/tasks/set_default_dashboard.yaml (hunk 3 above)
Comment (ShellyXueHan) [reply scripts/tasks/set_default_dashboard.yaml:37]: nope, the token exists by itself, it's not generated during the playbook.

File: scripts/main.yaml (hunk 1 above)
Comment (ShellyXueHan) [reply scripts/main.yaml:13]: you can help to improve the playbook by added in action variable and checks ;P

**Pull request merged**